src/Movie.js

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 import './Movie.css'
 
-const Movie = ({ id, poster_path, backdrop_path, title, average_rating, release_date }) => {
-  console.log(Math.floor(average_rating))
-
+const Movie = ({ id, poster_path, backdrop_path, title, average_rating, release_date, showProfile }) => {
   const rottenTomatillos = '🤢'
 
+  // const handleClick = (id) => {
+  //   const currentMovie = this.state.allMovies.find(movie => movie.id === id)
+  //   this.setState({ currentMovie: currentMovie })
+  // }
+
+  const showMovie = (id) => {
+    // const currentMovie = this.state.allMovies.find(movie => movie.id === id)
+    showProfile(id)
+    console.log(id)
+    // this.setState({ currentMovie: currentMovie })
+  }
+
   return (
-    <article id={id}>
+    <article id={id} onClick={() => showMovie(id)}>
```

Comment:
So this would read `onClick={() => showProfile(id)}`

Reply:
In order to check that this is working, use your React Dev Tools to check that App's state is updating to show a currentMovie.

Reply:
Thanks for all this feedback. I'll work on this today!!